src/Cache/Signature/FixerSignature.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Cache\Signature;
+
+use PhpCsFixer\Fixer\FixerInterface;
+
+final class FixerSignature
+{
+    private string $name;
+    private string $contentHash;
```

Comment:
I do believe that this is naive implementation and I am not feeling comfortable with it.

### problem 1
- imagine this hierarchy `AbstractFixer -> AbstractPhpDocFixer -> PhpDocReturnTypeFixer`
- have `contentHash` of `PhpDocReturnTypeFixer` , yet someone changed implementation of one of methods in `AbstractPhpDocFixer` - will `contentHash` change? I don't see that happening. And lack of `contentHash` change means that fixer will not run, even when it should.

### problem 2
imagine `FooFixer` using `BarService`, and we change `BarService` - again, `FooFixer` should run, but it won't.

### problem 3+
probably more to come

### solution?
we should likely get hash of everything `FooFixer` using, all the parent classes, traits, side classes, libraries, ...